optimizely/config/datafileprojectconfig/mappers/experiment.go

```diff
@@ -44,6 +44,11 @@ func mapVariation(rawVariation datafileEntities.Variation) entities.Variation {
 		Key:            rawVariation.Key,
 		FeatureEnabled: rawVariation.FeatureEnabled,
 	}
+
+	for _, variable := range rawVariation.Variables {
+		variation.Variables = append(variation.Variables, entities.VariationVariable{ID: variable.ID, Value: variable.Value})
```

Comment:
I think we can benefit from making `Feature.Variables` a map of string to variable for easier access. It also helps that we can easily iterate over maps in Go.